validators/notes.go

```diff
@@ -39,9 +39,16 @@ func ValidateDeleteNoteRequest(req *notespb.DeleteNoteRequest) error {
 }
 
 func ValidateListNoteRequest(req *notespb.ListNotesRequest) error {
-	return validation.ValidateStruct(req,
+	err := validation.ValidateStruct(req,
 		validation.Field(&req.AuthorAccountId, validation.Required),
 	)
+	if err == nil {
+		return err
+	}
+	err = validation.ValidateStruct(req,
+		validation.Field(&req.GroupId, validation.Required),
+	)
+	return err
 }
```

Comment:
`req.AuthorAccountId` and `req.GroupId` are not required! At least one has to be set but you can have

```
ListNotes(authorAccountId, groupId)
ListNotes(groupId)
ListNotes(authorAccountId)
```

Reply:
yes thats the point of this code,